plugins/pulp_rpm/plugins/distributors/yum/metadata/repomd.py

```diff
@@ -24,6 +24,8 @@ def __init__(self, working_dir, checksum_type=CONFIG_DEFAULT_CHECKSUM,
         super(RepomdXMLFileContext, self).__init__(metadata_file_path, checksum_type)
         self.gpg_sign = gpg_sign
         self.sign_options = sign_options
+        self.file_locations = []
```

Comment:
what about renaming to metadata_file_locations or some similar name to point what kind of files locations we are referencing to?

Reply:
yep, that can be done